templates/docs/examples/patterns/tooltips/_tooltips_script.js

```diff
@@ -0,0 +1,34 @@
+function handleTooltipEvents(tooltip) {
+  var trigger = document.querySelector('[aria-describedby="' + tooltip.getAttribute('id') + '"]');
+
+  trigger.addEventListener('mouseenter', function () {
+    // remove this class on mouseenter,
+    // rather than unnecessarily on mousemove
+    tooltip.classList.remove('u-hide');
```

Comment:
In theory tooltip may show up in unexpected place if `mouseenter` is triggered, but `mousemove` is not.
Not sure if it may happen, but showing and positioning should happen together.
I think moving removing the hide class name to `mousemove` handler should work. Or on mouseenter it should also be positioned correctly.

Reply:
:+1: moved the setting of x and y coords to a separate function